cmd/neofs-adm/internal/modules/morph/config.go

```diff
@@ -223,3 +224,84 @@ func dumpNetworkConfig(cmd *cobra.Command, _ []string) error {
 func invalidConfigValueErr(key []byte) error {
 	return fmt.Errorf("invalid %s config value from netmap contract", key)
 }
+
+const (
+	configKeyFlag   = "key"
+	configValueFlag = "value"
+)
+
+func setConfigCmd(cmd *cobra.Command, _ []string) error {
+	wCtx, err := newInitializeContext(cmd, viper.GetViper())
+	if err != nil {
+		return fmt.Errorf("can't to initialize context: %w", err)
+	}
+
+	cs, err := wCtx.Client.GetContractStateByID(1)
+	if err != nil {
+		return fmt.Errorf("can't get NNS contract info: %w", err)
+	}
+
+	nmHash, err := nnsResolveHash(wCtx.Client, cs.Hash, netmapContract+".neofs")
+	if err != nil {
+		return fmt.Errorf("can't get netmap contract hash: %w", err)
+	}
+
+	k := viper.GetString(configKeyFlag)
+	if k == "" {
+		return errors.New("empty config key")
+	}
+
+	v := viper.GetString(configValueFlag)
+	if v == "" {
+		return fmt.Errorf("empty config value for %s key", k)
+	}
```

Comment:
Why did you choose to use `-k` and `-v`?
We already have a similar thing in `set-policy`, where we use a list of `key=value` pairs. This allows us to change multiple parameters in a single transaction.